compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/transform/GroupByKeyAndWindowDoFnTransform.java

```diff
@@ -144,7 +197,10 @@ public String toString() {
     return sb.toString();
   }
 
-  private Iterable<TimerInternals.TimerData> getTimers(final InMemoryTimerInternals timerInternals) {
+  /**
+   * Get timer data.
+   */
+  private List<TimerInternals.TimerData> getTimers(final InMemoryTimerInternals timerInternals) {
```

Comment:
getEligibleTimers?
getTriggeredTimers?
(It appears that we're collecting timers based on some properties)

Reply:
getEligibleTimers looks good